website/content/docs/k8s/connect/terminating-gateways.mdx

```diff
@@ -88,6 +88,58 @@ Registering the external services with Consul is a multi-step process:
 
 ### Register external services with Consul
 
+There are two ways to register an external service with Consul:
+1. If [`TransparentProxy`](/docs/k8s/helm#v-connectinject-transparentproxy) is enabled, you can declare external endpoints in the [`Destination`](/docs/connect/config-entries/service-defaults#terminating-gateway-destination) field of `service-defaults`.
+1. You can add the service as a node in the Consul catalog.
+
+#### Register an external service as a Destination
+
+`Destination` fields allow clients to dial the external service directly and are valid only in [`TransparentProxy`](/docs/k8s/helm#v-connectinject-transparentproxy) mode.
+The following table describes traffic behaviors when using `Destination`s to route traffic through a terminating gateway:
+
+| External Services Layer | Client dials | Client uses TLS | Allowed | Notes |
+|---|---|---|---|---|
+| L4 | Hostname | Yes | Allowed | `CAFiles` are not allowed because traffic is already end-to-end encrypted by the client. |
+| L4 | IP | Yes | Allowed | `CAFiles` are not allowed because traffic is already end-to-end encrypted by the client. |
+| L4 | Hostname | No | Not allowed | Because the Sidecar is not protocol aware, it will not be able to identify traffic going to the external service. |
+| L4 | IP | No | Allowed | There are no limitations on dialing IPs without TLS |
+| L7 | Hostname | Yes | Not allowed | Because traffic is already encrypted before the sidecar, it cannot route as L7 Traffic. |
+| L7 | IP | Yes | Not allowed | Because traffic is already encrypted before the sidecar, it cannot route as L7 Traffic. |
+| L7 | Hostname | No | Allowed | A `Host` or `:authority` header is required. |
+| L7 | IP | No | Allowed | There are no limitations on dialing IPs without TLS  |
```

Comment:
```suggestion
| L4 | Hostname | No | Not allowed | The sidecar is not protocol aware and can not identify traffic going to the external service. |
| L4 | IP | No | Allowed | There are no limitations on dialing IPs without TLS. |
| L7 | Hostname | Yes | Not allowed | Because traffic is already encrypted before the sidecar, it cannot route as L7 traffic. |
| L7 | IP | Yes | Not allowed | Because traffic is already encrypted before the sidecar, it cannot route as L7 traffic. |
| L7 | Hostname | No | Allowed | A `Host` or `:authority` header is required. |
| L7 | IP | No | Allowed | There are no limitations on dialing IPs without TLS.  |
```
Just a few super minor style things: use simple present tense and use lower case unless it's a proper name.